src/llmcompressor/observers/base.py

```
@@ -26,6 +27,12 @@ class QParamsDict(TypedDict, total=False):
     zero_point: torch.Tensor
     global_scale: Optional[torch.Tensor]
 
+class _CustomFP8ScaleData(FloatArgs):
+    exponent = 4
+    mantissa = 3
+    bits = 8
+    max = gs_max
```

Ruff (0.15.21)

[error] Undefined name `gs_max`

(F821)